src/workerd/api/streams/standard.h

```diff
@@ -375,7 +355,7 @@ class WritableImpl {
 
   struct Writable {};
 
-  kj::Maybe<WriterOwner&> owner;
+  kj::Own<WeakRef<WritableStreamJsController>> owner;
```

Comment:
this type is mentioned a lot. maybe typedef would be convenient?

Reply:
Perhaps, I'll save that for a future cleanup tho